tests/fun_provider.py

```diff
@@ -0,0 +1,15 @@
+from algorithms.color_refinement import color_refinement, fast_color_refinement
```

Comment:
Ik snap niet zo goed waarom deze er nog is, omdat we in de unittests niet willen kunnen kiezen voor andere algoritmen.

Misschien dat we ergens wel een overkoepelende methode color_refinement() kunnen hebben die door alle tests wordt aangeroepen en dat we daarin de color_refinement kunnen zetten die we willen gebruiken. Of dat nu de gewone is of de fast.

Reply:
Ik probeerde hier een simpele wrapper te maken die dat op 1 plek doet.
Maar wat jij voorstelt kan ook.
Kunnen we zoals @JKleinRot zegt in haar comment dit punt van testen eerst beter defineren voor deze PR verder behandelen.